lib/encryptor.rb

```diff
@@ -15,7 +15,10 @@ module Encryptor
   #
   # Run 'openssl list-cipher-commands' in your terminal to view a list all cipher algorithms that are supported on your platform
   def default_options
-    @default_options ||= { :algorithm => 'aes-256-cbc' }
+    @default_options ||= {
+        :algorithm => 'aes-256-cbc',
+        :hmac_algorithm => 'sha1',
```

Comment:
Initially I feared compatibility issues, since we rely on libopenssl for our digest impls. On closer examination it looks like SHA256 has been supported since the first release of 0.9.8, so I'm probably being overly cautious. Safe (and fast) defaults FTW; will change to sha256.